eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/ConfirmedDelivery.scala

```diff
@@ -62,12 +71,6 @@ trait ConfirmedDelivery extends EventsourcedActor {
   }
 
   /**
-   * Confirms the message delivery identified by `deliveryId`.
-   */
-  def confirm(deliveryId: String): Unit =
```

Comment:
The scala-doc on `deliver` still mentions `confirm` instead of `persistConfirmation`